### PR TITLE
fix: Responsive Landing Page

### LINKED ISSUE #156: Fix responsiveness of landing page
Currently when viewing the home page and default courses page, the `Created by students @ VikeLabs` text at the bottom will wrap text, which doesn't look great.

The first screenshot is how it should always look, and the ladder shows currently what happens.

### PR DESCRIPTION
**Pull request by FlyteWizard**
## Overview

Closes #156 

* Add responsive design to the landing page content.

The `VIKE LABS` part of the landing page is still open for changes. Feel free to give a better alternative or modifications.

NOTE: This does not fix the height + content issue, and only fixes the responsiveness. 

## Large Desktop

Capture d’écran, le 2021-06-04 à 21 02 54

## Medium Desktop 

Capture d’écran, le 2021-06-04 à 20 51 43

## Small Screen

Capture d’écran, le 2021-06-04 à 20 51 56

## Smallest Screen

Capture d’écran, le 2021-06-04 à 20 52 22

Known Issue:

This will not be fixed in this PR as it is more relevant to course scheduling page than landing page. This will need to be fixed once a suitable responsive design is created for this page.

Capture d’écran, le 2021-06-04 à 21 27 45

### CODE REVIEW

File: src/app/landing/index.tsx [hunk 1]
```diff
@@ -19,33 +19,35 @@ export default function Landing() {
...
+      <GridItem colSpan={1}>
+        <Container>
+          <Heading fontSize="4.75em" lineHeight="1.25" mb="1.75rem">
+            Explore UVic Courses
+          </Heading>
+          <Text fontSize="1.6em">CourseUp makes it simple to browse and schedule UVic Courses</Text>
+          <Flex alignItems="center" mt="1rem" as="a" w="fit-content" target="_blank" href="https://www.vikelabs.ca">
+            <Text color="#4C6EA5" fontSize={{ base: '1em', md: '1.5em' }}>
+              <span>{hands[handIndex]}</span> Built by students @
+              <span style={{ fontWeight: 'bolder', fontSize: '1.75em', color: '#222B49' }}> VIKE LABS</span>
```
Comment (keithradford) [inline src/app/landing/index.tsx:37]: style nit: splitting both these spans into separate `Text` attributes as children of the flexbox allow for the text to be more centered which IMO looks a bit better with the differing font sizes.

```ts
<Text color="#4C6EA5" fontSize={{ base: '1em', md: '1.5em' }}>
    <span>{hands[handIndex]}</span> Built by students @
</Text>
<Text fontWeight="bolder" fontSize="1.75em" color="#222B49" ml="1">
     VIKE LABS
</Text>
```


Screen Shot 2021-06-05 at 9 19 58 AM

Comment (FlyteWizard) [reply src/app/landing/index.tsx:37]: Good catch. 👍

**Comment by FlyteWizard** [comment]
@keithradford or @aomi Have you folks ever had an outside contribution? This is the second time it has failed, but it probably shouldn't be failing?

**Comment by keithradford** [comment]
@FlyteWizard Yeah, this is because it's an outside contribution. Your forked repo doesn't have access to the secrets the build needs. This specific build provides us with a [preview link](https://github.com/VikeLabs/courseup/pull/168#issuecomment-851227102), so that's what we're missing here. Not a _huge_ deal and isn't a blocker on the bright side.

**Comment by aomi** [comment]
@FlyteWizard I've removed that step for forks so it shouldn't "fail" now :+1:

**Review by keithradford — APPROVED** [review]
LGTM

**Pull request merged**